### PR TITLE
Added Julia HTTP.jl framework

### PR DESCRIPTION
**Pull request by sumeetchhetri**

### CODE REVIEW

**Comment by sumeetchhetri** [comment]
@nbrady-techempower, please review this PR as well.

File: .travis.yml [hunk 1]
```diff
@@ -58,6 +58,7 @@ env:
     - 'TESTDIR="JavaScript/0http JavaScript/express JavaScript/fastify JavaScript/hapi JavaScript/koa"'
     - 'TESTDIR="JavaScript/nodejs JavaScript/polkadot JavaScript/restana JavaScript/restify JavaScript/sailsjs"'
     - 'TESTDIR="JavaScript/es4x JavaScript/ringojs JavaScript/just"'
+    - 'TESTDIR="Julia/Http.jl"
```
Comment (nbrady-techempower) [inline .travis.yml:61]: Missing the closing `'` here, but you can change this to:

```
    - "TESTLANG=Julia"
```

**Comment by nbrady-techempower** [comment]
This looks good @sumeetchhetri. Just needs a README.md when you have a chance

**Comment by sumeetchhetri** [comment]
> This looks good @sumeetchhetri. Just needs a README.md when you have a chance

Added a short README

**Pull request merged**